<?xml version="1.0"?>
<launch>

  <arg name="UAV_NAME"   default="$(env UAV_NAME)" /> 
  <!-- default is realsense -->
  <arg name="camera_type"   default="rgbd" /> 

    <group ns="$(arg UAV_NAME)">

      <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>

      <node pkg="nodelet" type="nodelet" name="LandoltNodelet" args="load capra/LandoltNodelet standalone_nodelet" output="screen">
        <!-- <param name="camera_reading" type="str" value="/$(arg UAV_NAME)/$(arg camera_type)/color/image_raw" /> -->
        <!-- <param name="camera_basler" type="str" value="/$(arg UAV_NAME)/basler_down/image_raw" /> -->
        <param name="camera_reading" type="str" value="/$(arg UAV_NAME)/pico_flexx_left/image_mono16" />
        <!-- <param name="camera_picoflexx_right" type="str" value="/$(arg UAV_NAME)/pico_flexx_right/image_mono16" /> -->
        <!-- <param name="camera_picoflexx_back" type="str" value="/$(arg UAV_NAME)/pico_flexx_back/image_mono16" /> -->
        <!-- <param name="camera_realsense" type="str" value="/$(arg UAV_NAME)/rgbd/color/image_raw" /> -->
        <!-- <param name="camera_oakd" type="str" value="/$(arg UAV_NAME)/oak/rgb/image_raw" /> -->
      </node>

    </group>
</launch>

